librz/core/cmd.c

```diff
@@ -2163,6 +2157,59 @@ RZ_API int rz_core_cmd_pipe(RzCore *core, char *rizin_cmd, char *shell_cmd) {
 	return ret;
 }
 
+static char *system_exec_stdin(int argc, char **argv, const ut8 *input, int input_len, int *length) {
+	char *output = NULL;
+	if (!rz_subprocess_init()) {
+		RZ_LOG_ERROR("Cannot initialize subprocess.\n");
+		return NULL;
+	}
+
+	RzSubprocessOpt opt = {
+		.file = argv[0],
+		.args = (const char **)&argv[1],
+		.args_size = argc - 1,
+		.envvars = NULL,
```

Comment:
do we want to pass some vars to the subprocess? `RZ_FILE`, `RZ_OFFSET`, `RZ_ENDIAN`, etc.?

Reply:
hmm i do not know.

Reply:
according to `!??` programs are expected to find some env vars. It seems that in the past implementation that was all set with `rz_core_sysenv_begin`/`rz_core_sysenv_end`. It should be fine to keep this as-is, because current implementation of RzSubprocess just allows to add/replace env variables, but not to remove them. So by setting `.envvars = NULL`  we are really just keeping the parent env vars, which should be fine.